backends/apple/mps/mps_preprocess.py

```diff
@@ -2,12 +2,13 @@
 #  Copyright (c) 2023 Apple Inc. All rights reserved.
 #  Provided subject to the LICENSE file in the top level directory.
 #
-
 import logging
-from typing import Dict, final, List
+from typing import ClassVar, Dict, final, List, Tuple
 
 import torch
 
+from backends.apple.mps.serialization.mps_graph_schema import Buffer
```

Comment:
Could you add this change? The test is failing internally...
```suggestion
from executorch.backends.apple.mps.serialization.mps_graph_schema import Buffer
```

Reply:
@cccclai @DenisVieriu97  Do we know why it's not consistent? Make it fail on GitHub is better in this case